arm/Microsoft.ApiManagement/service/deploy.bicep

```diff
@@ -224,34 +194,211 @@ resource apiManagementService 'Microsoft.ApiManagement/service@2020-12-01' = {
     apiVersionConstraint: !empty(minApiVersion) ? json('{"minApiVersion": "${minApiVersion}"}') : null
     restore: restore
   }
-  resource apiManagementService_signin 'portalsettings@2019-12-01' = if (!empty(portalSignIn)) {
-    name: 'signin'
-    properties: portalSignIn
-  }
-  resource apiManagementService_signup 'portalsettings@2019-12-01' = if (!empty(portalSignUp)) {
-    name: 'signup'
-    properties: portalSignUp
+}
+
+module apis_resource 'apis/deploy.bicep' = [for (api, index) in apis: {
+  name: '${uniqueString(deployment().name, location)}-api-${index}'
+  params: {
+    apiManagementServiceName: name
```

Comment:
change to `apiManagementServiceName: apiManagementService.name` and remove the dependency.

The same for all other resources

Reply:
done

Reply:
please remove the dependency too. It's now implicit

Reply:
done